Source/Core/Math/V2.h

```diff
@@ -0,0 +1,118 @@
+#pragma once
+#include "Debug/DebugHelpers.h" 
+
+#include <DirectXMath.h>
+#include <cmath>
+
+
+struct V2 {
+	float x, y;
```

Comment:
Make these be separate line and initialize them in the header
```suggestion
	float x = 0.0f;
	float y = 0.0f;
```